File: app/src/main/java/in/woloo/www/my_account/mvp/MyAccountView.java
package in.woloo.www.my_account.mvp;

import in.woloo.www.more.models.UserCoinHistoryModel;
import in.woloo.www.more.models.UserCoinsResponse;

public interface MyAccountView {
    void userCoinsSuccess(UserCoinsResponse userCoinsResponse);
    void userCoinsHistorySuccess(UserCoinHistoryModel userCoinHistoryModel);


    /*
            calling on userCoinsHistorySuccess
        */
   // void userCoinsHistorySuccess(UserCoinHistoryModel userCoinHistoryModel);
}

File: app/src/main/java/in/woloo/www/more/my_account/mvp/MyAccountView.kt
package `in`.woloo.www.more.my_account.mvp

import `in`.woloo.www.more.models.UserCoinHistoryModel
import `in`.woloo.www.more.models.UserCoinsResponse

interface MyAccountView {
    fun userCoinsSuccess(userCoinsResponse: UserCoinsResponse?)
    fun userCoinsHistorySuccess(userCoinHistoryModel: UserCoinHistoryModel?) /*
            calling on userCoinsHistorySuccess
        */
    // void userCoinsHistorySuccess(UserCoinHistoryModel userCoinHistoryModel);
}
